src/util/simd/sse2.rs

```diff
@@ -1,11 +1,26 @@
 use std::{
     arch::x86_64::*,
     ops::{BitAnd, BitOr, BitOrAssign},
+    sync::Once,
 };
 
 use super::{Mask, Simd};
 use crate::impl_lanes;
 
+#[inline]
```

Comment:
We do not need the optimization.  The generated asm from ` std::arch::is_x86_feature_detected` has optimizations. 
https://rust.godbolt.org/z/sdqefTPxW

Reply:
Ah! That's nice to know. I'll be reverting that then